dvc/remote/ssh/connection.py

```diff
@@ -183,14 +168,13 @@ def remove(self, path):
             self._remove_file(path)
 
     def download(self, src, dest, no_progress_bar=False, progress_title=None):
-        if no_progress_bar:
-            self.sftp.get(src, dest)
-        else:
-            if not progress_title:
-                progress_title = os.path.basename(src)
+        if not progress_title:
+            progress_title = os.path.basename(src)
```

Comment:
Strictly speaking this should not be called if `no_progress_bar is True`.

Reply:
negligible overhead and better readability? I don't mind either way

Reply:
negligible overhead and better readability? I don't mind either way

Reply:
going to mark this as resolved unless someone until someone expresses a strong opinion